SU2_IDE/Xcode/SU2_CFD.xcodeproj/project.pbxproj

```diff
@@ -717,8 +795,18 @@
 				GCC_WARN_UNUSED_PARAMETER = NO;
 				GCC_WARN_UNUSED_VALUE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = /opt/intel/mkl/include;
 				INSTALL_PATH = /usr/local/bin;
-				MACOSX_DEPLOYMENT_TARGET = 10.11;
+				LIBRARY_SEARCH_PATHS = /opt/intel/mkl/lib;
+				MACOSX_DEPLOYMENT_TARGET = 10.13;
+				OTHER_CPLUSPLUSFLAGS = "-DHAVE_MKL";
```

Comment:
-DHAVE_MKL is not working in Xcode on my machine, since I do not have it installed. Prob best to leave this off by default, so the project always builds.

Reply:
Tom, I will disable the MKL options.